generate.sh

```diff
@@ -119,50 +109,6 @@ EOF
     yq -e ea '.bundles | .[]' "$cfg" | xargs -n1 printf '    - image: %s@%s\n' "$bundle_reg_from"
 } > "$render_config"
 
-echo "-> Defining supported OCP versions" >&2
```

Comment:
This is pulling definition of the catalogs/contributions schemas for the composite templates.  Instead, we just uniformly apply the semver expansion to all OCP_VER.

Reply:
Makes sense